.github/workflows/master.yml

```diff
@@ -183,6 +183,12 @@ jobs:
       with:
         name: test-results-${{ matrix.modules }}-${{ matrix.comment }}-${{ matrix.java }}-${{ matrix.hadoop }}-${{ matrix.hive }}
         path: "**/target/test-reports/*.xml"
+    - name: Upload unit tests log files
+      if: failure()
```

Comment:
Zipped logfiles are ~300 MB. Jenkins always uploads but GitHub Actions uploads failed tests only for now.

I can't find the clear answer of the size limit from my googling but seems like there's a limit internally. Should be best to save the resources.

Reply:
Logs for failed tests only should be good enough from my experience.

Reply:
With `if: failure()`, only the failed test logs are supposed to be uploaded, right?

Reply:
Yup yup.